src/index.html

```diff
@@ -12,78 +12,67 @@ <h1>List of employees</h1>
       <li
         data-position="Accountant"
         data-salary="$162,700"
-        data-age="33"
-      >
+        data-age="33">
```

Comment:
it isn't a correct fix for tag attributes

Reply:
Sorry, that was autoformatting probably, observed it only after push.